<launch>
  <!-- VRPN Server -->
  <arg name="server" default="192.169.1.11"/>
  <!-- <arg name="server" default="mocap.local"/> -->

  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
      <!-- server: $(arg server) -->
      server: $(arg server)
      port: 3883
      update_frequency: 100.0
      frame_id: map
      use_server_time: false
      broadcast_tf: true
      refresh_tracker_frequency: 1.0
    </rosparam>

   
    <!-- ARMLab rigidbody Remap -->
    <remap from="/vrpn_client_node/Locobot1/pose" to="camera_frame/mavros/vision_pose/pose" />
    <remap from="/vrpn_client_node/box/pose" to="box/mavros/vision_pose/pose" />
    <remap from="/vrpn_client_node/apriltag/pose" to="apriltag/mavros/vision_pose/pose" />

  </node>
</launch>
